beacon-chain/state/state-native/getters_state.go

```diff
@@ -222,6 +222,51 @@ func (b *BeaconState) ToProtoUnsafe() interface{} {
 			PendingPartialWithdrawals:     b.pendingPartialWithdrawals,
 			PendingConsolidations:         b.pendingConsolidations,
 		}
+	case version.Fulu:
+		lookahead := make([]uint64, len(b.proposerLookahead))
+		for i, v := range b.proposerLookahead {
```

Comment:
I don't think we need to do this if the proto was cast correctly in the first place